otelcol/testdata/otel-log-to-file.yaml

```diff
@@ -14,7 +14,7 @@ exporters:
 service:
   telemetry:
     logs:
-      level: warn
+      level: info
```

Comment:
This is to make the Windows service test pass. Before this change, we had the warning from confighttp and configgrpc, but after it there are no logs at warn level with this config, which makes the assertion that the logs were stored in a file fail